test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/004_ATF_N_TC_existing_navi_app_certificate_exist.lua

```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] SDL must start PTU for navi app right after app successfully registration
+--
+-- Description:
+-- In case navigation app connects and sucessfully registers on SDL (opens RPC 7 service)
+-- and PolicyTable has NO "certificate" at "module_config" section of LocalPolicyTable
+-- SDL must start PolicyTableUpdate process on sending SDL.OnStatusUpdate(UPDATE_NEEDED) to HMI to get "certificate"
+--
+-- 1. Used preconditions:
+-- Navi app exists in LP, certificate is listed in module_config
+--
+-- 2. Performed steps
+-- Register navi application.
+--
+-- Expected result:
+-- Application is registered successfully.
+-- SDL should not trigger PTU. SDL.OnStatusUpdate(UPDATE_NEEDED) should not be sent.
```

Comment:
@istoimenova,
Seems description contradicts to preconditions ('no certificate' vs 'certificate is listed')
Could you add the following statement from req-t to description: "In case "certificate" exists at "module_config" section -> SDL must NOT trigger PTU but start TLSHandshake"